.github/workflows/sync_labels.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow synchronizes groups of labels that correspond
+# to items of selection list in Trac. It controls that in each
+# such case there is just one label of the list present.
+# Furthermore in the case of the state it checks the labels
+# to coincide with the corresponding review state.
+
+name: Synchronize selection list lables
+
```

Comment:
I cannot read this name well. We don't need to remind us about "the selection list in Trac" as we are now in GitHub. 
Perhaps we may say that the script synchronizes the labels with the state of the PR.

How about just "Synchronize labels" for the name, and edit the comment as well?

Reply:
Now I understand why you say "selection list labels". But I guess this means nothing to those who don't know the origin of the labels.

Reply:
Done!